### PR TITLE
New module - meraki_nat

### LINKED ISSUE #52874: New module - meraki_nat
##### SUMMARY
Meraki added support for configuring NAT rules via the API. A `meraki_nat` module should be created to allow configuration.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
meraki

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
Meraki added support for NAT objects in the API. This module allows for querying and manipulating 1:1 rules, 1:many rules, and port forwarding rules.

~~Depends on https://github.com/ansible/ansible/pull/48394~~ (merged)

Fixes #52874

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
meraki_nat

### CODE REVIEW

**Comment by kbreit** [comment]
ready_for_review

File: lib/ansible/modules/network/meraki/meraki_nat.py [hunk 1]
```diff
@@ -0,0 +1,679 @@
...
+    net_name:
+        description:
+        - Name of a network.
+        aliases: [name, network]
+        type: str
+    net_id:
+        description:
+        - ID number of a network.
+        type: str
+    org_name:
```
Comment (Qalthos) [inline lib/ansible/modules/network/meraki/meraki_nat.py]: `auth_key`, `org_name`, and `org_id` are already documented in the meraki doc_fragment, and should not need to be specified here.

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/meraki/meraki_nat.py (hunk 1 above)
Comment (kbreit) [reply lib/ansible/modules/network/meraki/meraki_nat.py]: I have an inconsistency in the type of `org_id` between the specs and documentation. Meraki announced they're changing the type to a string in the coming weeks anyways, so I will be doing a separate PR across all modules to make `org_id` consistent and correct. The others have been fixed.

**Comment by kbreit** [comment]
@Qalthos I've updated this one based on your suggestions. Please review.

**Review by pabelanger — APPROVED** [review]

**Review by Qalthos — APPROVED** [review]

**Pull request merged**